### PR TITLE
Targets issue number #217 add support for StopIfTrue to conditional f…

### PR DESCRIPTION
**Pull request by NickNack2020**
Targets Issue #217 - Add support for StopIfTrue for conditional Formats.

### CODE REVIEW

**Comment by igitur** [comment]
Thanks. Please check out the failing tests.

**Comment by NickNack2020** [comment]
@igitur The test have been fixed. I adjusted the code so I do not serialize the default value which will keep the files smaller, and avoid me from having to reconciling all the existing tests.

Shame on me for not running all the tests the first time.

**Comment by igitur** [comment]
Thanks. Also look at the review comment.

**Comment by NickNack2020** [comment]
@igitur which code review comments are you referring to? I do not see any. Am I missing something?

File: ClosedXML_Examples/ConditionalFormatting/ConditionalFormatting.cs [hunk 1]
```diff
@@ -634,4 +634,30 @@ public void Create(String filePath)
...
+        {
+            var workbook = new XLWorkbook();
+            var ws = workbook.AddWorksheet("Sheet1");
+
+            ws.FirstCell().SetValue(6)
+                .CellBelow().SetValue(1)
+                .CellBelow().SetValue(2)
+                .CellBelow().SetValue(3);
+
+            var cf = ws.RangeUsed().AddConditionalFormat();
```
Comment (NickNack2020) [inline ClosedXML_Examples/ConditionalFormatting/ConditionalFormatting.cs]: I can make that change. Thanks .

Comment (NickNack2020) [inline ClosedXML_Examples/ConditionalFormatting/ConditionalFormatting.cs]: It should be noted that percent and bottom are not exposed this way, but I can make the change requested.

**Comment by NickNack2020** [comment]
@igitur requested changes have been made.

**Comment by igitur** [comment]
Sorry, I've neglected this. Please pull in the new `develop` and resolve the conflicts.

**Comment by igitur** [comment]
@NickNack2020 Ping...

**Comment by NickNack2020** [comment]
Please reject this PR and use https://github.com/ClosedXML/ClosedXML/pull/501 instead. The merge conflict was all over the place since I had originally forked before we dropped 3.5 support. It was cleaner to do it this way.

**Pull request closed without merging**